operator-common/src/main/java/io/strimzi/operator/common/metrics/MetricsUtils.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.common.metrics;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.Tags;
+import io.strimzi.operator.common.model.Labels;
+import org.apache.logging.log4j.util.Strings;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Utility class for metrics
+ */
+public class MetricsUtils {
+    /**
+     * Private constructor to prevent instantiation
+     */
+    private MetricsUtils() {
+        // Intentionally left blank
+    }
+
+    /**
+     * Tag representing any namespace
+     */
+    protected static final Tag TAG_ANY_NAMESPACE = Tag.of("namespace", "*");
```

Comment:
Should we use `ConfigParameter.ANY_NAMESPACE` instead of the hardcoded `*`?

Reply:
Sure 😄